app/View/ApiUsers/fields.inc

```diff
@@ -52,17 +60,24 @@
 <?php endif; // co_id == 1 ?>
 <ul id="<?php print $this->action; ?>_api_users" class="fields form-list">
   <li>
-    <div class="field-name vtop">
+    <div class="field-name">
       <div class="field-title">
         <?php print ($e ? $this->Form->label('username', _txt('fd.api.username')) : _txt('fd.api.username')); ?>
         <span class="required">*</span>
       </div>
       <div class="field-desc"><?php print _txt('fd.api.username.desc'); ?></div>
     </div>
     <div class="field-info">
-      <?php print ($e
-        ? $this->Form->input('username', array('class' => 'focusFirst', 'default' => $cur_co['Co']['name'].'.'))
-        : filter_var($api_users[0]['ApiUser']['username'],FILTER_SANITIZE_SPECIAL_CHARS)); ?>
+      <?php
+        $prefix = 'co_' . $cur_co['Co']['id'] . '.';
+        print ($e ? $this->Form->input('username', array(
+                                                    'class' => 'focusFirst',
+                                                    'label' => $prefix,
```

Comment:
The use of the $prefix as the label here is clever, but probably not best from an accessibility standpoint. We have a label in the field-name above, so rather than produce the prefex as one as well simply wrap it with a span and print it out directly, e.g. `print '<span class="api-username-prefix">' . $prefix . '</span>';`

Reply:
I made the change.